### ISSUE TITLE
Unexpected critical error: Line number must be positive integer. Got: 0

### ISSUE THREAD

**Issue by JanMikes** [DaveLiddament/sarb#30]
Hello!

We had following error reported by Phpstan:
```
ProfilePresenter.php:0:ParseError (syntax error, unexpected end of file, expecting function (T_FUNCTION) or const (T_CONST)) thrown while autoloading class Klarka\TagHide\TagHide.
```

I understand what the error says and it is completely legitimate. Though when running SARB: `vendor/bin/sarb remove-baseline-results reports/phpstseline.json reports/errors_since_baseline.txt`

There is following error: `Unexpected critical error: Line number must be positive integer. Got: 0`

I think SARB should allow number `0` in line, because it is legit error and should be reported.

What do you think?

**Comment by DaveLiddament** [comment]
Yes I'll have a look at allowing line number of 0 (essentially this represents and issues with the file rather than a line of code). 

In the interim would adding linting as a step before PHPStan help. I use `jakub-onderka/php-parallel-lint` Maybe this would catch the kind of problem and fix it before running PHPStan?

**Comment by DaveLiddament** [comment]
Released in 0.3.3 https://github.com/DaveLiddament/sarb/releases/tag/0.3.3

**Comment by JanMikes** [comment]
Hi @DaveLiddament thank you very much for implementing, it makes a lot of sense to me :+1: 

We have managed to implement parallel-lint into our CI, so we can discover this with more detailed info.

Though i was surprised if the tool (phpstan) supports it, that SARB ends up with critical error :-).
